examples/abitest/module_0/rust/src/lib.rs

```diff
@@ -80,6 +80,19 @@ impl FrontendNode {
             oak::channel_close(wh.handle).unwrap();
             oak::channel_close(rh.handle).unwrap();
         }
+
+        {
+            // Create a new node to handle HTTP requests
+            info!("Starting HTTP server pseudo-node on port 8383.");
```

Comment:
Could you make `[::]:8383` a const and use it both as part of the log message and the `init` call, so that we do not accidentally change only one of them in the future? (also print the actual address, not just the port)